ui/components/app/snaps/snap-delineator/snap-delineator.js

```diff
@@ -27,10 +30,16 @@ import {
 export const SnapDelineator = ({
   snapName,
   type = DelineatorType.default,
+  isCollapsable = false,
+  isCollapsed = true,
   children,
+  onClick,
+  boxProps,
 }) => {
   const t = useI18nContext();
-  const isError = type === DelineatorType.Error;
+  const isError =
+    type === DelineatorType.Error || type === DelineatorType.Warning;
```

Comment:
Do we not want separate styling for warnings?

Reply:
The styling was using the same as the error state so I thought it would be redundant to define an `isWarning` or something for the same thing. I can come back and do that once I refine the styling more post-hackathon.